readinglist/views/article.py

```diff
@@ -118,6 +119,7 @@ def preprocess_record(self, new, old=None):
 
         return new
 
+    @resource.view(permission='readwrite')
```

Comment:
you should add it to put() as well. It should not return 405 if the user is not authenticated